ci/helm/values/values-mongodb.yaml.gotmpl

```diff
@@ -1,6 +1,11 @@
 image:
   tag: {{ .Values.mongodb.image.tag }}
+{{ if eq .Environment.Name "functional-tests-2023" }}
+auth:
+  enabled: false
+{{ else }}
 usePassword: false
+{{ end}}
```

Comment:
This switch should be extracted to two files, one for the mongodb4 release and another one for mongdb6 releases.
This is about maintainability, and ease future environments. Furthermore these new files will be used on release enabled condition and not environment name like currently.